runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -84,9 +84,11 @@
 		"Your amulet of chemistry has (\\d) charges? left\\."
 	);
 	private static final Pattern AMULET_OF_CHEMISTRY_USED_PATTERN = Pattern.compile(
-		"Your amulet of chemistry helps you create a 4-dose potion\\. (?:<col=ff0000>)?It has (\\d|one) charges? left\\."
+		"Your amulet of chemistry helps you create a \\d-dose potion\\. (?:<col=ff0000>)?It has (\\d|one) charges? left\\."
```

Comment:
To test the number of doses, don't we have to use a capture group? The plugin only captures the number of charges.
```suggestion
		"Your amulet of chemistry helps you create a (\\d)-dose potion\\. (?:<col=ff0000>)?It has (\\d|one) charges? left\\."
```